dmipy/core/modeling_framework.py

```diff
@@ -594,12 +604,37 @@ def set_tortuous_parameter(self, lambda_perp_parameter_name,
                     param)
                 raise ValueError(msg)
 
+        if S0_intra is None and S0_extra is None:
+            S0_intra = 1.
+            S0_extra = 1.
+        elif S0_intra is not None and S0_extra is not None:
```

Comment:
So if I understand correctly, if someone sets both `S0_intra` and `S0_extra` then these values must correspond with values in `self.S0_tissue_responses`. 

It seems a bit superfluous to actually have to give the response values if they have to match anyway.

To simplify the interface, what do you think of having a boolean like `use_s0_responses_if_given` in `set_tortuous_parameter`, and then infer which S0s should be used from the `volume_fraction_intra_parameter_name` and `volume_fraction_extra_parameter_name`? the names are ordered in the same order as S0_responses so it should work.

Reply:
That's a good idea. Done :)